Sources/Kitura/String+Extensions.swift

```diff
@@ -20,9 +20,20 @@ import LoggerAPI
 /// String Utils
 extension String {
 
-    /// Parses percent encoded string into query parameters
+    /// Parses percent encoded string into query parameters with comma-separated
+    /// values.
     var urlDecodedFieldValuePairs: [String : String] {
```

Comment:
Although this does now look like a slow path (it uses `urlDecodedFieldMultiValuePairs` then joins them with commas), I suspect it may be faster than before as we no longer do string interpolation which is known to be slow.

Having said that it would be nice to know what the performance impact is. @NocturnalSolutions did you measure any difference? @djones6 do you have any perf tests that would cover this?

Reply:
https://github.com/wg/wrk is pretty good - point it at a Kitura app with a route that exercises this codepath...